src/quo2/components/reacts.cljs

```diff
@@ -1,50 +1,66 @@
-                                       {:margin-top 25
-                                        :border-width 1
-                                        :border-color (if dark?
-                                                        colors/white-opa-5
-                                                        colors/neutral-80)})}
-   [icons/icon :main-icons/add-reaction-emoji48
-    {:color (if dark?
-              colors/white
-              colors/black)}]])
+  [{:keys [on-press]}]
+  (let [dark? (theme/dark?)]
+    [rn/touchable-opacity {:on-press on-press
+                           :style (merge reaction-styling
+                                         {:margin-top 25
+                                          :border-width 1
+                                          :border-color (if dark?
+                                                          colors/white-opa-5
+                                                          colors/neutral-80)})}
+     [icons/icon :main-icons/add-reaction20
+      {:color (if dark?
+                colors/white
+                colors/black)}]]))
 
 (defn render-react
   "Add your emoji as a param here"
-  [{:keys [emoji clicks dark? neutral? on-press]}]
-  (let [text-color (if dark? colors/white
+  [{:keys [emoji clicks neutral? on-press]}]
+  (let [dark? (theme/dark?)
+        text-color (if dark? colors/white
                        colors/black)
         numeric-value (int clicks)
         clicks-positive? (pos? numeric-value)]
     [rn/touchable-opacity {:on-press on-press
                            :style (merge reaction-styling
                                          (cond-> {:background-color
                                                   (if dark?
-                                                    (if neutral? colors/neutral-70 colors/neutral-90)
-                                                    (if neutral? colors/neutral-30 colors/neutral-10))}
+                                                    (if neutral?
+                                                      colors/neutral-70
+                                                      "transparent")
+                                                    (if neutral?
+                                                      colors/neutral-30
+                                                      "transparent"))}
                                            (and dark? (not neutral?)) (assoc :border-color colors/neutral-70
                                                                              :border-width 1)
                                            (and (not dark?) (not neutral?)) (assoc :border-color colors/neutral-30
                                                                                    :border-width 1)))}
-     [quo/text {:style {:color text-color}}
-      (str emoji (if clicks-positive?
-                   (str " " numeric-value)
-                   ""))]]))
+     [icons/icon emoji {:color "default"
```

Comment:
what is "default" color ?

Reply:
I'll nil so that it uses the default color

Reply:
I just need to add the emoji's default color and not color it at all
If I didn't add the color attribute, it gets colored white

Reply:
`:no-color true`

Reply:
Thanks Andre
Done